blockchain_storage/src/mock.rs

```diff
@@ -48,38 +51,60 @@ mockall::mock! {
         fn del_block_id_at_height(&mut self, height: &BlockHeight) -> crate::Result<()>;
     }
 
-    impl storage::traits::Transactional<'_> for Store {
-        type TransactionRo = MockStoreTx;
-        type TransactionRw = MockStoreTx;
-        fn start_transaction_ro(&self) -> MockStoreTx;
-        fn start_transaction_rw(&self) -> MockStoreTx;
+    impl<'tx> crate::Transactional<'tx> for Store {
```

Comment:
My issue is `Transactional` now I don't really have a good reason to dislike it (maybe it sounds weird as it's an adjective?) but I do. But as I said before I can deal with it